lib/dialects/mysql/schema/columncompiler.js

```diff
@@ -74,7 +72,7 @@ Object.assign(ColumnCompiler_MySQL.prototype, {
   },
 
   datetime(precision) {
-    if (isObject(precision)) {
+    if (precision instanceof Object) {
```

Comment:
can you create an internal isObject function and refer to it from everywhere, so that it would be easier to replace underlying implementation in the future? there are certain edge cases for this check.

Reply:
i.e. Object.create(null)

Reply:
That also works